sdcm/nemesis.py

```diff
@@ -656,7 +671,45 @@ def disrupt_truncate(self):
         # do the actual truncation
         self.target_node.run_cqlsh(cmd='TRUNCATE {}.{}'.format(keyspace_truncate, table), timeout=120)
 
-    def _modify_table_property(self, name, val, filter_out_table_with_counter=False):
+    def _get_compaction_strategy(self, node, keyspace, table):
+        """Get a given table compaction strategy
+
+        Arguments:
+            node {str} -- ip of db_node
+            keyspace
+            table
+        """
+        list_tables_compaction = node.run_cqlsh('SELECT keyspace_name, table_name, compaction FROM system_schema.tables',
```

Comment:
There is a function called "get_non_system_ks_cf_list" in common.py, you could use that.
Anyway, you need to filter the system tables in advance.